native-image-tests/build.gradle.kts

```diff
@@ -38,14 +39,12 @@ animalsniffer {
 }
 
 sourceSets {
-  // Not included in IDE as this confuses Intellij for obvious reasons.
   main {
     java.srcDirs(
-      "../okhttp/src/test/java",
```

Comment:
These tests have started to rely on internals which break this approach.  We should consider pulling out our tests into a test fixtures module or similar.  Less graal test coverage for now.